build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+N_CORES=$(sysctl -n hw.ncpu) # macOS
+# TODO: get cores linux
+
+# ESP-IDF v3.x
+esp_idf_v3_build_native() {
+    cp arduinojason.mk components/ArduinoJson/component.mk
+    make all -j$N_CORES;
+    rm components/ArduinoJson/component.mk
+}
+
+# build docker v3.x
+esp_idf_v3_build_docker() {
+    N_CORES_DOCKER=$(docker run --rm espressif/idf:v3.3.4 /bin/bash -c "grep processor /proc/cpuinfo | wc -l; exit" | tail -1)
+    cp arduinojason.mk components/ArduinoJson/component.mk
```

Comment:
What do you think about a one-liner of "echo 'COMPONENT_ADD_INCLUDEDIRS := src' > components/ArduinoJson/component.mk" instead of having this file?

Reply:
Funny. I made this change before even seeing this comment.